server/routes/request.js

```diff
@@ -23,4 +24,6 @@ router
   .route("/:id")
   .patch(validateId, protect, validateCreateUpdateRequest, updateRequest);
 
+router.route("/v2/").post(protect, createRequestSitter);
```

Comment:
```suggestion
router.route("/").post(protect, createRequestSitter);
```

Any reason why we are creating a duplicate route here? Also, consider adding the `validateCreateUpdateRequest` middleware to this route